server/sonar-web/src/main/js/apps/issues/workspace-list-view.js

```diff
@@ -45,35 +45,35 @@ export default WorkspaceListView.extend({
       selectedIssueView.$('.js-issue-' + action).click();
     };
     WorkspaceListView.prototype.bindShortcuts.apply(this, arguments);
-    key('right', 'list', function () {
+    key('right', 'list', () => {
       const selectedIssue = that.collection.at(that.options.app.state.get('selectedIndex'));
       that.options.app.controller.showComponentViewer(selectedIssue);
       return false;
     });
-    key('space', 'list', function () {
+    key('space', 'list', () => {
       const selectedIssue = that.collection.at(that.options.app.state.get('selectedIndex'));
       selectedIssue.set({ selected: !selectedIssue.get('selected') });
       return false;
     });
-    key('f', 'list', function () {
+    key('f', 'list', () => {
       return doAction('transition');
     });
-    key('a', 'list', function () {
+    key('a', 'list', () => {
       return doAction('assign');
     });
-    key('m', 'list', function () {
+    key('m', 'list', () => {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove curly braces and "return" from this arrow function body. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)